README.rst

```diff
@@ -50,31 +48,29 @@ Also included are SQL drivers:
 |Hibernate5        |Java 8 or 11         |
 +------------------+---------------------+
 
-Of course, these clients require a NuoDB database to be running.  If you don't
-have NuoDB_ available yet, you can use the `NuoDB Community Edition`_.
+The use of the client drivers require a running NuoDB database.  If you don't
+have a NuoDB_ database running in your environment yet, you can use the NuoDB `Community Edition`_ page to guide you through the steps to deploy a NuoDB database.
 
 Installation
 ------------
 
-Unpack the tar file, or unzip the zip file.  No other installation is
-required.
+Unpack the tar file, or unzip the zip file into your chosen NuoDB Client package installation directory.
```

Comment:
This is a bit misleading because the tar/zip file contains its own directory.  This makes it sound like you have to pre-create some kind of root directory, like `/opt/nuodb` or whatever.